src/Http/Validation/Domain.php

```diff
@@ -53,4 +55,32 @@ protected function stripProtocol($domain)
 
         return $domain;
     }
+
+    /**
+     * Strip the port from a domain.
+     *
+     * @param $domain
+     *
+     * @return mixed
+     */
+    protected function stripPort($domain)
+    {
+        $domainStripped = preg_replace(self::PATTERN_STRIP_PROTOCOL, '', $domain);
```

Comment:
But in fact, I should have been a bit more verbose in my previous comment... does it really makes sense to have `$domainStripped`.
We coud achieve exactly the same doing:
```php
return preg_replace('/:\d*$/', null, $domain);
```